### PR TITLE
Add Mem.ai Extension

### PR DESCRIPTION
**Pull request by brianfakhoury**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by brianfakhoury** [comment]
Hi @pernielsentikaer! Sorry to bother you -- I just want to check on the status of this request as I want to continue developing the extension once I know an initial deployment is available.

Thanks for your consideration!

File: extensions/mem/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm onSubmit={handleSubmit} title="Create mem" />
+        </ActionPanel>
+      }
+    >
+      <Form.TextArea placeholder="Start typing your mem here..." {...itemProps.content} />
+      <Form.Separator />
+      <Form.DatePicker title="Schedule for later?" {...itemProps.scheduledFor} />
```
Comment (pernielsentikaer) [inline extensions/mem/src/index.tsx:55]:
```suggestion
      <Form.DatePicker title="Schedule for Later?" {...itemProps.scheduledFor} />
```

File: extensions/mem/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <Form.Checkbox title="Mark as read?" label="Read" {...itemProps.isRead} />
```
Comment (pernielsentikaer) [inline extensions/mem/src/index.tsx:56]:
```suggestion
      <Form.Checkbox title="Mark as Read?" label="Read" {...itemProps.isRead} />
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* It seems to work when I don't add a date, if I add a date I get this error:

```
08:05:55.748 ClientRequestError: Request failed with error code: INTERNAL_SERVER_ERROR
    at Object._handleGraphqlRequestErrors [as handleRequestErrors] (/Users/pernielsentikaer/.config/raycast/extensions/mem/index.js:13933:17)
    at MemClient2.<anonymous> (/Users/pernielsentikaer/.config/raycast/extensions/mem/index.js:14788:38)
    at Generator.throw (<anonymous>)
    at rejected (/Users/pernielsentikaer/.config/raycast/extensions/mem/index.js:14735:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by brianfakhoury** [comment]
Thanks, @pernielsentikaer for the review! It seems like their API might be an issue, so let me double-check if I need to disable that feature temporarily.

**Comment by brianfakhoury** [comment]
@pernielsentikaer I was able to resolve the bug! I believe it's ready for deployment now.

I also enabled the drafts feature for the form for better UX!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**